spec/system/admin/order_spec.rb

```diff
@@ -1023,6 +1023,7 @@ def new_order_with_distribution(distributor, order_cycle)
         it "ships the order and shipment email is sent" do
           expect(page).to have_content "ready"
           expect(page).not_to have_content "shipped"
+          expect(page).to have_css("#s2id_add_variant_id")
```

Comment:
This is a bit unclear as to what you are trying to do. We have some custom "[select2" matcher](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/support/matchers/select2_matchers.rb)

So you can do something like bellow which is a lot easier to understand : 
```suggestion
          expect(page).to have_select2 "add_variant_id"
```